CHANGELOG.rst

```diff
@@ -20,19 +20,21 @@ This document describes changes between each past release.
 **Breaking changes**
 
 - Renamed internal backend classes for better consistency. Settings remain
-  unchanged (#).
+  unchanged (#491).
 - ``cliquet.schema`` is now deprecated, and was moved to a ``cliquet.resource``
-  module.
-- Resource collection attribute is now deprecated. Use ``model`` attribute instead.
+  module. (#505)
+- Resource collection attribute is now deprecated. Use ``model`` attribute instead. (#506)
 
 **Internal changes**
 
 - Rework PostgreSQL backends to use composition instead of inheritance for the
-  client code.
-- Replace DROP INDEX by a conditional creation in PostgreSQL schemas (#487,
+  client code. (#491)
+- Replace DROP INDEX by a conditional creation in PostgreSQL schemas (#487, #496
   thanks @rodo)
-- Documentation and minor refactors in viewset code
+- Documentation and minor refactors in viewset code (#490, #498, #502
 - Add the ``build-requirements``, ``distclean`` and ``maintainer-clean`` Makefile rules.
```

Comment:
I realize that there is no doc for the last two (at least to know when to use them)